site/tailwind.config.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
                     .filter((i) => i)
                     .reduce((acc, i) => ({ ...acc, [i]: `${i / (baseFontSize / 4)}rem` }), {}),
             }),
+            backgroundImage: {
+                "govuk-crest": "url('../public/assets/images/govuk-crest-2x.png')",
```

Comment:
We can use the absolute path here `/assets/images/govuk-crest-2x.png` since the relative path might not work when the site is deployed